src/extension-test.ts

```diff
@@ -0,0 +1,136 @@
+/**
+    Copyright 2023 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import {
+  RunnerExtension,
+  createRunner,
+  parse,
+  stringify,
+} from '../lib/main.js';
+import { importExtensionFromPath } from './CLIUtils.js';
+import http from 'http';
```

Comment:
With my comment bellow we should stick to one convention for now. In the feature (After EOF of Node 14) we should use only `node:${pacakage}` .